modules/contentbox/models/system/CBHelper.cfc

```diff
@@ -2563,4 +2563,17 @@ component accessors="true" singleton threadSafe {
 		return b.toString();
 	}
 
+	/**
+	 * Convenience method to ensure valid JSON, when prefixing is enabled
+	 *
+	 * @obj any  the object to be serialized
+	 */
+	string function toJSON( any obj ){
```

Comment:
ColdBOx already has a `forAttribute()` function to deal with CF -> JS apps. I think this is not necesasry

Reply:
I would prefer we update the `forAttribute()` function in the framework itself.

Reply:
Actually `forAttribute` does not work for this use case, as it also runs the JSON through `encodeForHTMLAttribute`, which is not what we want in this case - as it is being presented in a `script` block.  If we do modify the core, I would suggest adding this method, and then have `forAttribute` use the new method for its JSON

Reply:
Created: https://ortussolutions.atlassian.net/browse/COLDBOX-1159

Reply:
Updated to use new `toJSON` method in supertype/Util and bumped Coldbox to `be` until the next version is released.